tvtk/pyface/ui/qt4/QVTKRenderWindowInteractor.py

```diff
@@ -271,50 +177,17 @@ def _get_win_id(self):
     def Finalize(self):
```

Comment:
IIRC the finalize used to cause a crash on OS X, I can't remember exactly how right now but can you test this version please to ensure that when you create a window and close it, it doesn't crash.  Thanks!